### DISCUSSION TITLE
Backend Services

### DISCUSSION THREAD

**Discussion by AirTechWick** [theta360developers/thanks#30] [Ideas]
Custom Front-End UI for Google Forms as the Backend
HTML Data -> Google Forms
https://www.youtube.com/watch?v=_gsZbIIMS7w

**Comment by AirTechWick** [comment]
Custom Thank you page for Google Forms
https://www.youtube.com/watch?v=Q8t9CqdIY2Q

**Reply by codetricity** [reply]
Nice job getting this working.

If we move to Google Forms, I would prefer to look into the officially supported API for Google Forms.

https://developers.googleblog.com/2022/03/introducing-google-forms-api.html

https://developers.google.com/forms/api/reference/rest

There are many tutorials on YouTube to use the official API. 

It looks like the Google Forms API was just released March 17, 2022

I suggest we spend time looking at it and building a test site with the API and move this site to the Google Forms API.

**Comment by AirTechWick** [comment]
From the Blog -> Google Forms API gives access to:
- Managing Forms
- Acting on Responses
- ### Build integrations on top of Forms 
( Which is probably what we need to integrate Google Forms into a web page )
- Enables automated form creation and editing 
- Enables developers to **build automations for acting on incoming responses**

**Comment by AirTechWick** [comment]
This is a faster more concise version of the First Video I sent above made by another person, does the same thing and follows the same method does HTML Data ->Google Forms
https://www.youtube.com/watch?v=0udw0nol6Po

**Comment by AirTechWick** [comment]
Article Version of the HTML Data -> Google Forms method
Doesn't use API still
https://medium.com/geekculture/link-your-html-form-to-a-spreadsheet-via-google-forms-9024f0611d82

**Comment by codetricity** [comment]
Maybe it's not possible to submit the contents of a HTML form to the Google Form using the API?

https://stackoverflow.com/questions/71714110/can-you-submit-a-restful-request-to-a-google-forms-api
